package.json

```diff
@@ -559,6 +564,19 @@
       }
     ],
     "menus": {
+      "editor/context": [
+        {
+          "submenu": "microsoft-powerapps-portals.powerpages-copilot",
+          "group": "0_powerpages-copilot",
+          "when": "never"
```

Comment:
This is set to not show up at all at the moment right?

Reply:
right, have added the configuration...will take implementation in separate PR